### ISSUE TITLE
Requiring two converges is stupid

### ISSUE THREAD

**Issue by werebus** [umts/chef-kedpm-transit#1]
[This](https://github.com/umts/chef-kedpm-transit/blob/master/recipes/sysadmins.rb#L36) is a bit of a hack.  Because of the compile-time / converge-time problem, sysadmins that _will_ be created this time through don't get their config file.

The attempt to use the `/etc/skel/` directory as a work-around doesn't even work, because this happens:
1.  Compile-time happens and `Etc.getpwuid` can't find the uid yet
2.  Users are created by the users cookbook
3.  Config file is dropped off by our recipe (too late)

There must be a better way right?

**Comment by werebus** [comment]
Maybe `Etc.getpwuid` is too low-level for this stuff.  Is it OK to just assume that the sysadmins will all have a home directory of `/home/username/`?

**Comment by werebus** [comment]
Sort of...

https://github.com/opscode-cookbooks/users/blob/master/providers/manage.rb#L68-L81

The users databag might have a home directory specified.  So something like 

``` ruby
sa['username'] ||= sa['uid']
home = sa['home'] || "/home/#{sa['username']}/"
```

should work.

**Comment by werebus** [comment]
Fixed by 421ee4d5bb8594e750a1651215349c34efad5166
